torch/nn/modules/loss.py

```diff
@@ -771,13 +771,15 @@ class SmoothL1Loss(_Loss):
     .. math::
         z_{i} =
         \begin{cases}
-        0.5 (x_i - y_i)^2, & \text{if } |x_i - y_i| < 1 \\
-        |x_i - y_i| - 0.5, & \text{otherwise }
+        0.5 (x_i - y_i)^2 / beta, & \text{if } |x_i - y_i| < beta \\
```

Comment:
the above isn't accurate anymore ""Creates a criterion that uses a squared term if the absolute
    element-wise error falls below 1 and an L1 term otherwise.	    element-wise error falls below 1 and an L1 term otherwise."

Reply:
we should also document the beta < 0.